settings.py

```diff
@@ -2205,3 +2208,23 @@ def _determine_couch_databases(couch_databases):
             'NAME': 'custom.nic_compliance.password_validation.UsedPasswordValidator',
         }
     ]
+
+
+RCH_PERMITTED_FIELD_MAPPINGS = {
+    'mother': json.load(open(os.path.join('custom', 'rch', 'all_fields', 'mother.json')))['fields'],
+    'child': json.load(open(os.path.join('custom', 'rch', 'all_fields', 'child.json')))['fields']
+}
+
+
+def extract_rch_fields_from_mapping(beneficiary_type):
+    field_mappings = RCH_PERMITTED_FIELD_MAPPINGS[beneficiary_type]
+    rch_fields = []
+    for case_type in field_mappings:
+        rch_fields = rch_fields + field_mappings[case_type].keys()
+    return rch_fields
+
+
+RCH_PERMITTED_FIELDS = {
+    'mother': extract_rch_fields_from_mapping('mother'),
+    'child': extract_rch_fields_from_mapping('child'),
+}
```

Comment:
would be nice to make these lazy so that they don't slow down the startup of every Django process